analyzers/tests/SonarAnalyzer.UnitTest/Helpers/NetFrameworkVersionProviderTest.cs

```diff
@@ -122,7 +122,7 @@ private static Compilation GetRawCompilation(IEnumerable<MetadataReference> theR
         private static string CreateMockPath(string mockName)
         {
             var assembly = typeof(NetFrameworkVersionProviderTest).Assembly;
-            return Path.Combine(Path.GetDirectoryName(assembly.Location), PathResolver.RelativePathToTestProjectRoot, "../FrameworkMocks/lib/", mockName);
+            return Path.Combine(Path.GetDirectoryName(assembly.Location), Paths.TestProjectRoot, "../FrameworkMocks/lib/", mockName);
```

Comment:
All of these usages should not need their prefixes now
```suggestion
            return Path.Combine(Paths.TestProjectRoot, "../FrameworkMocks/lib/", mockName);
```

Reply:
That's sooo much better and consistent now :)